controller/src/main/java/org/jboss/as/controller/audit/ManagedAuditLoggerImpl.java

```diff
@@ -528,6 +528,35 @@ public void updateInMemoryHandlerMaxHistory(String name, int maxHistory) {
         }
     }
 
+    public void copy(ManagedAuditLoggerImpl other) throws IOException {
```

Comment:
If the process is booting and the audit logger is not configured for boot logging, I think this should just not copy.